schemas/ncbi/ncbi_taxon.yaml

```diff
@@ -1,5 +1,10 @@
 name: ncbi_taxon
```

Comment:
So can we ditch the canonicalization of names in the nodes then?

Reply:
Yeah, most likely